Core/Src/CAN/inverter_can.c

```diff
@@ -63,8 +67,10 @@ void CAN_inverter_receive_callback(FDCAN_HandleTypeDef* hfdcan, uint32_t RxFifo0
             /* Reception Error */
             Error_Handler();
         }
-        osThreadFlagsSet(t_inverter_comm_errorHandle, INVERTER_CAN_ACTIVE);
+    osMessageQueuePut(q_ids_can_inverterHandle, &hfdcan1, 0, osWaitForever); //colocar timeout
```

Comment:
Seguindo a ideia de usar o Header no lugar do hfdcan.
```suggestion
    uint32_t id = RxHeader.Identifier;
    osMessageQueuePut(q_ids_can_inverterHandle, &id, 0, osWaitForever); 
```